### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by elizoller**
**GitHub Issue**: (https://github.com/Islandora-CLAW/CLAW/issues/1267)

[Tech call minutes](https://github.com/Islandora-CLAW/CLAW/wiki/September-25%2C-2019#minutes)

# What does this Pull Request do?
Updates Readme to standards

# What's new?
An updated readme and logo

# How should this be tested?

Give it a read and compare it to the [template](https://github.com/Islandora-CLAW/CLAW/blob/master/docs/contributing/readme_template.md).

# Interested parties
@Islandora-CLAW/committers @manez

### CODE REVIEW

**Comment by elizoller** [comment]
Screen Shot 2019-09-27 at 10 48 07 AM
opensea-lobster

**Comment by elizoller** [comment]
I updated the license to link to the one in the repo and it appears to be the right "kind" of license for Drupal module but I can't actually speak to the content of the License file.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # OpenSeadragon
+# ![Mascot](https://user-images.githubusercontent.com/5439169/65790675-0242b600-e115-11e9-817f-e31c41bf2ece.png)
```
Comment (manez) [inline README.md]: Image should go before the repo name so they end up on the same lie: `# Alt text Repository Name`

File: README.md [hunk 2]
```diff
@@ -7,19 +8,58 @@
...
+
+## Documentation
+
+Further documentation for this module is available on the [Islandora 8 documentation site](https://islandora-claw.github.io/CLAW/).
+
+## Troubleshooting/Issues
+
+Having problems or solved a problem? Check out the Islandora google groups for a solution.
+
+* [Islandora Group](https://groups.google.com/forum/?hl=en&fromgroups#!forum/islandora) * [Islandora Dev Group](https://groups.google.com/forum/?hl=en&fromgroups#!forum/islandora-dev)
```
Comment (manez) [inline README.md]: should be two bullets instead of one line of text. This may be a problem carried over from the template?

File: README.md [hunk 3]
```diff
@@ -7,19 +8,58 @@
...
+
 ## Maintainers
 
 Current maintainers:
 
 * [Jared Whiklo](https://github.com/whikloj)
```
Comment (manez) [inline README.md]: Should remove the old maintainer. It's all you now, @elizoller 😄

File: README.md [hunk 4]
```diff
@@ -7,19 +8,58 @@
...
+If you would like to contribute code to the project, you need to be covered by an Islandora Foundation [Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_cla.pdf) or [Corporate Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_ccla.pdf). Please see the [Contributors](http://islandora.ca/resources/contributors) pages on Islandora.ca for more information.
+
+We recommend using the [claw-playbook](https://github.com/Islandora-Devops/claw-playbook) to get started. If you want to pull down the submodules for development, don't forget to run `git submodule update --init --recursive` after cloning.
 
 ## License
 
-[GPLv2](http://www.gnu.org/licenses/gpl-2.0.txt)
+[GPLv2](https://github.com/Islandora-CLAW/openseadragon/blob/8.x-1.x/LICENSE)
```
Comment (manez) [inline README.md]: This link works, but I think it can be simplified to `[GPLv2](/LICENSE]` ?

**Pull request merged**